_docs/3x/tutorials/parallelism.md

```diff
@@ -31,15 +31,12 @@ We can do parallel execution in batches, that does deterministic
 ### The Naive Way
 
 The following example uses
-[Task.gather]({{ page.path | api_base_url }}monix/eval/Task$.html#gather[A,M[X]<:TraversableOnce[X]](in:M[monix.eval.Task[A]])(implicitcbf:scala.collection.generic.CanBuildFrom[M[monix.eval.Task[A]],A,M[A]]):monix.eval.Task[M[A]]),
+[Task.parSequence]({{ page.path | api_base_url }}monix/eval/Task$.htmll#parSequenceN[A](parallelism:Int)(in:Iterable[monix.eval.Task[A]]):monix.eval.Task[List[A]]),
 which does parallel processing while preserving result ordering, 
 but in order to ensure that parallel processing actually happens,
```

Comment:
I would remove this part too, `executeAsync` is not used in the example

Reply:
`but in order to ensure that parallel processing actually happens, the tasks need to be effectively asynchronous.` 
 but introducing the `.executeAsync` signature is good to reinforce the previous statement, isn't it? 
maybe there should be an example using it?

Reply:
`parSequence` will start all tasks asynchronously anyway so it doesn't matter if there's `executeAsync`, or not

Examples of `executeAsync` would be good for Concurrency-related section - I am working on one for `monix-bio` and it could be ported later